incubator/fluentd-cloudwatch/templates/daemonset.yaml

```diff
@@ -1,4 +1,8 @@
-apiVersion: extensions/v1beta1
+{{- if .Capabilities.APIVersions.Has "apps/v1" }}
+apiVersion: apps/v1
+{{- else }}
+apiVersion: apps/v1beta1
```

Comment:
shouldn't that be extensions/v1beta1?

Reply:
Hi @marcatos is there anything I can help with on this?

Reply:
Hi @davidkarlsen @sionsmith, sorry for this huge delay, I've been busy this month. 
I've just pushed a commit to fix my copy/paste error.